nodejsalert-ui/config.js

```diff
@@ -0,0 +1,36 @@
+var config = {
+development: {
+
+
+    database: {
+        host:   'localhost',
+        user:   'root',
+        password: 'Hameed@123',
+        database: 'sampledb'
+    },
+    //3scale details
+    threescale: {
+         url: 'https://3scale-prod-1.app.rhdp.ocp.cloud.lab.eng.bos.redhat.com:443/cicd/maingateway/',
+         token: 'ab6283abd6fd695fd4b31e8bf82d5e462c13e998'
+    }
+},
+production: {
+
+
+    //mongodb connection settings
+    database: {
```

Comment:
Please make sure these are all configurable and I think these dbuser / password shouldn't saved here but should be parameter placeholders

Reply:
The parameters are configurable, and config.js file is main configuration file for node.js project. However I removed  local development SQL credential from file.